### ISSUE TITLE
2019-11-22 tc010, tc013, tc015, tc016 FLGACCEPTKEL == 0 for patients when OPTIMIZEKEL == 1

### ISSUE THREAD

**Issue by jhhughes256** [tensfeldt/openNCA#55]
### Issue

When running tc010, tc013, tc015 or tc016 with OPTIMIZEKEL == 1 all subjects are given a FLGACCEPTKEL value of zero, regardless of whether they meet the criteria. This does not occur for tc012.

### Example

Below I have provided output for tc010. I rounded AUCXPCTP to six decimals so that it wouldn't appear in scientific notation.

```
> r$AUCXPCTP <- round(r$AUCXPCTP, 6)
> any(r$FLGACCEPTKEL == 1)
[1] FALSE
> r[r$FLGACCEPTKEL == 0, c("SDEID", "KEL", "KELTMLO", "KELTMHI", "KELNOPT", "KELRSQ", "AUCXPCTP", "FLGACCEPTKEL")]
       SDEID        KEL KELTMLO KELTMHI KELNOPT    KELRSQ  AUCXPCTP FLGACCEPTKEL
1  111299859 0.05815238    12.0      48       4 0.9947858  6.818886            0
2  112146587 0.08695843     8.0      48       5 0.9906898  1.839371            0
3  123811839 0.16211187     2.0      16       5 0.9977814  0.000000            0
4  145593586 0.04150156     8.0     120       7 0.9954984  0.600945            0
5  202367199 0.05406080    12.0      96       6 0.9912606  0.100089            0
6  233891605 0.05680128    16.0      96       5 0.9967403  0.058987            0
7  257914734 0.03141471     1.5      72       6 0.9827484  8.987003            0
8    2925168 0.04240636    12.0      72       5 0.9980178  4.983491            0
9  355441192 0.04656816    12.0     120       7 0.9943149  0.266796            0
10  40876220 0.04126708    12.0      72       5 0.9609512  5.543119            0
```

**Comment by rudraya** [comment]
This issue was addressed in 18821dc, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
### Update

This issue no longer occurs as of 18821dc.
